src/test/java/spoon/test/pkg/PackageTest.java

```diff
@@ -43,7 +46,7 @@ public void testPackage() throws Exception {
 		Assert.assertEquals(packageInfoFile.getCanonicalPath(), ctPackage.getPosition().getFile().getCanonicalPath());
 		Assert.assertEquals(1, ctPackage.getPosition().getLine());
 		Assert.assertEquals(1, ctPackage.getAnnotations().size());
-		Assert.assertEquals("This is test\nJavaDoc.", ctPackage.getDocComment());
```

Comment:
See the PR of @tdurieux about comments: https://github.com/INRIA/spoon/pull/554/files#diff-072d4b8ceab9c4e090f675ba9342b6f9L2050

A work has been done to remove usage of this method.